ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationSidebar.tsx

```diff
@@ -74,6 +74,22 @@ function isActiveLink(pathname: string, { isActive, path }: LinkDescription) {
     return typeof isActive === 'function' ? isActive(pathname) : Boolean(matchPath(pathname, path));
 }
 
+// Predicate function that encapsulates the logic for rendering content based on whether or not Vulnerability
+// Management 2.0 is declared as GA.
+function vulnMgmt2GaContentPredicate(
```

Comment:
I know this is specific to VM in this otherwise general file, but it made more sense to use a temporary util here instead of repeating the logic 4+ times in the declarative config below.